jquery-sortable-lists.js

```diff
@@ -43,6 +43,7 @@
 				},
 				opener: {
 					active: false,
+					as: 'img',
```

Comment:
Now I see that if we leave it without as and you will check it you must do
```
If ( o.as && o.as === 'html' ) 
```
So there are two conditions but I would like to have one 
Solution is to have as as default and check 
```
if ( o.as ==== 'html' )
```
or param will be optional like asHtm: true and so condition will be
```
If ( o.asHtml ) 
```
I would like to have that last option.